Create initial schema sql scripts for hwsc-user-svc

kimlisa opened the pull request.

kimlisa committed:
Create initial schema sql scripts for  hwsc-user-svc

kimlisa committed:
Add is_verified column to user_account table

kimlisa committed:
Remove password length constraint

faraonc commented on int/hwsc-user-svc/test_create.sh:
```diff
@@ -0,0 +1,15 @@
+docker exec -it devtest psql -U postgres -f /sql/S1_CREATE_DOMAINS.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S2_CREATE_USER_TABLE.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S3_CREATE_DOCUMENT_TABLE.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S4_CREATE_SHARED_TABLE.sql;\
```
I do not think this script is still needed. The script names are also wrong.

kimlisa replied:
Updated the script, this script is just for fast validation that table creation and inserts work as expected and view the table afterwards. Should I move it to gitignore?

faraonc replied:
we can ignore this file.

faraonc commented on int/hwsc-user-svc/test_insert.sql:
```diff
@@ -0,0 +1,12 @@
+INSERT INTO user_account (uuid, first_name, last_name, email, password, organization, create_timestamp)
```
The schema is different now.

kimlisa replied:
I set defaults so I don't have to set it in user-service, unless i have to for some reason:
is_verified should always be false on insert
created_date is defaulted to current date on insert time

should i be explicitly setting it in user-svc instead?

faraonc replied:
We should always manually set values in the service. Relying on a db makes it hard to debug later on.

kimlisa committed:
Update user_account table and test_insert

- Replaced Timestamp to Date
- Renamed create_timestamp to created_date
- Removed is_public default
- Updated test_insert.sql

kimlisa committed:
Update gitignore and test_insert.sql

- removed test_create.sh from master

faraonc commented:
TU, you can merge without another reviewer.
